### ISSUE TITLE
Restore the loss scale?

### ISSUE THREAD

**Issue by haochenglouis** [jiawei-ren/BalancedMSE#16]
Deat authors,

Thanks for the great job. I have a question:
why loss = loss * (2 * noise_var).detach(). can restore the loss scale?

**Comment by jiawei-ren** [comment]
We use the negative log of Gaussian as $\log\sigma_\text{noise} + (y-y_\text{pred})^2/{2\sigma_\text{noise}^2}$ ([code here](https://github.com/jiawei-ren/BalancedMSE/blob/main/imdb-wiki-dir/balanaced_mse.py#L22)), whereas the standard MSE is $(y-y_\text{pred})^2$. Thus multiplying $2\sigma_\text{noise}^2$ will keep them at the same scale.
